### PR TITLE
docs: Consolidate "Getting started" pages into a single guide

### PR DESCRIPTION
**Pull request by mnapoli**
As discussed internally, we have multiple "Getting started" guides today:

- Docs / [Get Started](https://www.serverless.com/framework/docs/getting-started)
- Docs / [Provider / AWS / User Guide / Quick start](https://www.serverless.com/framework/docs/providers/aws/guide/quick-start)
- Docs / [Provider / AWS / User Guide / Installation](https://www.serverless.com/framework/docs/providers/aws/guide/installation)

We want to simplify the onboarding experience with a single guide. This PR:

- Removes all "Getting started" guides except the 1st: Docs / [Get Started](https://www.serverless.com/framework/docs/getting-started)
- Moves the content of the other guides into the 1st: it's mostly about introducing main `serverless` commands, like deploy, remove, etc.
- Makes the "Getting started" guide more linear:
  - Install
  - Update
  - Setup
  - Main commands
  - Remove

After merging this, we will need the following redirects:

https://www.serverless.com/framework/docs/providers/aws/guide/quick-start -> https://www.serverless.com/framework/docs/getting-started
https://www.serverless.com/framework/docs/providers/aws/guide/installation -> https://www.serverless.com/framework/docs/getting-started

### CODE REVIEW

**Review by skierkowski — DISMISSED** [review]
Content LGTM.

**Review by pgrzesik — DISMISSED** [review]
Looks good 👍

File: docs/getting-started.md [hunk 1]
```diff
@@ -59,54 +58,103 @@ Install with [Chocolatey](https://chocolatey.org/):
...
 ```bash
+# Create a new serverless project
 serverless
+
+# Move into the newly created directory
+cd your-service-name
 ```
 
-The command will guide you to create a new serverless project.
+The `serverless` command will guide you to create a new project, [configure your AWS credentials](https://serverless.com/framework/docs/providers/aws/guide/credentials/), and optionally set up a free [Serverless Dashboard](https://www.serverless.com/monitoring) account to monitor, troubleshoot, and test your new service.
```
Comment (medikoo) [inline docs/getting-started.md:97]: Are we staying with _Dashboard_ name, or switching to _Console_  naming (I'm now really confused :) ?

Comment (mnapoli) [reply docs/getting-started.md:97]: Console is in private preview, readers of this documentation cannot sign up and use it rightaway.

File: docs/getting-started.md [hunk 2]
```diff
@@ -59,54 +58,103 @@ Install with [Chocolatey](https://chocolatey.org/):
 choco install serverless
 ```
 
-## Initial setup
+## Upgrade
```
[the remark this answers is not in the document]
Comment (mnapoli) [reply docs/getting-started.md:61]: No this is intentional, it goes like this:

- Install
  - Install as binary (optional)
- Upgrade
  - Upgrade standalone binary
- Getting started
- ...

The goal is to make the default method more visible, and secondary method (standalone binary) less important, it's a sub-section.

File: docs/getting-started.md [hunk 3]
```diff
@@ -19,21 +18,21 @@ menuItems:
...
+## Installation
 
 Install the serverless CLI via NPM:
 
 ```bash
 npm install -g serverless
 ```
 
-_Note: If you don’t already have [Node](https://nodejs.org/en/download/package-manager/) on your machine, install it first. We suggest using the latest LTS version of NodeJS._
+_Note: If you don’t already have [Node](https://nodejs.org/en/download/package-manager/) on your machine, install it first (for example [via Homebrew](https://formulae.brew.sh/formula/node)). We suggest using the latest LTS version of NodeJS._
```
[the remark this answers is not in the document]
Comment (mnapoli) [reply docs/getting-started.md]: The Slack discussion was about a homebrew install migrated from an Intel Mac to a M1 Mac (which isn't compatible). Homebrew and Node now support M1 today, but to be safe I updated this.

File: docs/getting-started.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (medikoo) [reply docs/getting-started.md:61]: Ok, I see that, looks good then 👍

File: docs/getting-started.md (hunk 1 above)
Comment (medikoo) [reply docs/getting-started.md:97]: Thanks for clarification :+1

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**